unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -38,19 +39,23 @@ export class CreatorLock extends React.Component {
       <LockIconBar lock={lock} toggleCode={this.toggleEmbedCode} />
     </LockIconBarContainer>)
 
+    let status
```

Comment:
hum. why do you need `status`? Maybe get rid of it since it is not being used anymore?